drivers/hmis/app/graphql/concerns/graphql_application_helper.rb

```diff
@@ -26,6 +26,10 @@ def access_denied!(message = 'access denied')
     raise message
   end
 
+  def policy_for(resource, policy:)
```

Comment:
Probably too late to change this, but passing a policy to a method called `policy_for` seems weird.  I _think_ this should probably be called `access_rights_for(resource, policy:)`.

Reply:
good point, how about:
`policy_for(referral, type: :ce_referral)`

(note ideally we should reshape our warehouse policy_for() to match)